src/customJs/payuButtonTool/index.test.jsx

```diff
@@ -0,0 +1,40 @@
+import { setLocale } from '../localization';
+import {PayuButtonTool} from "./PayuButtonTool";
+import * as React from "react"
+import {render, screen} from "@testing-library/react";
+
+describe(PayuButtonTool.name, ()=>{
+    it.each([
+        {baseUrl: 'https://cdn.fromdoppler.com/unlayer-editor/assets', size: 'small', lang: 'es-ES'},
+        {baseUrl: 'https://cdn.fromdoppler.com/unlayer-editor/assets', size: 'medium', lang: 'es-ES'},
+        {baseUrl: 'https://cdn.fromdoppler.com/unlayer-editor/assets', size: 'large', lang: 'es-ES'},
+        {baseUrl: 'https://cdn.fromdoppler.com/unlayer-editor/assets', size: 'small', lang: 'en-US'},
+        {baseUrl: 'https://cdn.fromdoppler.com/unlayer-editor/assets', size: 'medium', lang: 'en-US'},
+        {baseUrl: 'https://cdn.fromdoppler.com/unlayer-editor/assets', size: 'large', lang: 'en-US'}
+    ])("must be render with $size size and language $lang", ({baseUrl, size, lang})=>{
```

Comment:
I think that this setup would simplify the test logic:

```suggestion
    it.each([
        {size: 'small', lang: 'es-ES', 'https://cdn.fromdoppler.com/unlayer-editor/assets/payu-button-small-es.png`},
        {size: 'medium', lang: 'es-ES', 'https://cdn.fromdoppler.com/unlayer-editor/assets/payu-button-medium-es.png`},
        {size: 'large', lang: 'es-ES', 'https://cdn.fromdoppler.com/unlayer-editor/assets/payu-button-large-es.png`},
        {size: 'small', lang: 'en-US', 'https://cdn.fromdoppler.com/unlayer-editor/assets/payu-button-small-en.png`},
        {size: 'medium', lang: 'en-US', 'https://cdn.fromdoppler.com/unlayer-editor/assets/payu-button-medium-en.png`},
        {size: 'large', lang: 'en-US', 'https://cdn.fromdoppler.com/unlayer-editor/assets/payu-button-large-en.png`},
    ])("must be render with $size size and language $lang", ({size, lang, expectedImageUrl})=>{
```